app.js

```diff
@@ -1,4 +1,8 @@
 "use strict"
 
+require("./lib/database");
+
 const express = require("express");
 const app = express();
+
+app.listen(3000,()=> console.log("started"));
```

Comment:
read this port from .env file. It will cause problem when you want to deploy. You will have to keep editing app.js. Do something like this 

app.set('port', process.env.PORT || 3000);

Reply:
Alright